test/sql/insert_single.sql

```diff
@@ -35,3 +35,28 @@ INSERT INTO "3dim" VALUES('2017-01-20T09:00:47', 25.1, 'yellow', 'la');
 --queries should work in three dimensions
 SELECT * FROM "3dim";
 
+-- test that explain works
+EXPLAIN
+INSERT INTO "3dim" VALUES('2017-01-21T09:00:01', 32.9, 'green', 'nyc'),
+                         ('2017-01-21T09:00:47', 27.3, 'purple', 'la') RETURNING *;
+
+EXPLAIN
+WITH "3dim_insert" AS (
+     INSERT INTO "3dim" VALUES('2017-01-21T09:01:44', 19.3, 'black', 'la') RETURNING time, temp
+), regular_insert AS (
+   INSERT INTO regular_table VALUES('2017-01-21T10:00:51', 14.3) RETURNING time, temp
+) INSERT INTO "1dim" (SELECT time, temp FROM "3dim_insert" UNION SELECT time, temp FROM regular_insert);
+
+-- test prepared statement INSERT
+PREPARE "1dim_plan" (timestamp, float) AS
+INSERT INTO "1dim" VALUES($1, $2) ON CONFLICT (time) DO NOTHING;
+EXECUTE "1dim_plan" ('2017-04-17 23:35', 31.4);
+EXECUTE "1dim_plan" ('2017-04-17 23:35', 32.6);
+
+-- test prepared statement with generic plan (forced when no parameters)
+PREPARE "1dim_plan_generic" AS
+INSERT INTO "1dim" VALUES('2017-05-18 17:24', 18.3);
+EXECUTE "1dim_plan_generic";
```

Comment:
Can we have two EXECUTEs here. The first one would create the generic, the second one would execute the generic